EarlGrey.xcodeproj/xcshareddata/xcschemes/EarlGrey.xcscheme

```diff
@@ -26,16 +26,37 @@
       buildConfiguration = "Debug"
       selectedDebuggerIdentifier = "Xcode.DebuggerFoundation.Debugger.LLDB"
       selectedLauncherIdentifier = "Xcode.DebuggerFoundation.Launcher.LLDB"
+      language = ""
```

Comment:
Are these changes intentional or just a side-effect of opening in a different Xcode?